src/components/Form/Checkbox.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import styled from 'styled-components';
+import Label from './Label';
+
+const Container = styled.div`
+  position: relative;
+  color: ${props => props.theme.form.label.color};
+  background-color: ${props => props.theme.form.label.backgroundColor};
+  border-color: ${props => props.theme.form.label.borderColor};
+`;
+
+const StyledCheckbox = styled.input.attrs({
+  type: 'checkbox',
+  checked: props => props.checked,
+  disabled: props => props.disabled
+})`
+  opacity: 0;
+  width: 0;
+  height: 0;
+  margin: 0;
+
+  & + label {
+    font-size: ${props => props.theme.form.label.fontSize};
+    line-height: ${props => props.theme.form.label.fontSize};
+    font-weight: ${props => props.theme.form.label.fontWeight};
+    padding-left: 2.5rem;
+    cursor: pointer;
+
+    &:before,
+    &:after {
+      display: inline-block;
+      position: absolute;
+      left: 0;
+      top: 0.4rem;
+      border: 1px solid #ccc;
+      border-radius: 3px;
+      border-color: ${props => props.theme.form.checkbox.borderColor};
+      content: '';
+      width: 1.7rem;
+      height: 1.7rem;
+      transition: border 0.15s ease-in-out, color 0.15s ease-in-out;
+    }
+  }
+
+  &:checked + label:after {
+    font-family: FontAwesome;
+    content: '\F00C';
+  }
+
+  &:disabled + label,
+  &:disabled + label:after {
+    opacity: 0.3;
+    cursor: not-allowed;
+  }
+`;
+
+const Checkbox = ({ checked, disabled, className, ...props }) => {
+  const classes = classnames(className);
```

Comment:
Why call classnames here? Just let the `className` be catched in the props rest ;)

Reply:
Yup, one doubt though.
since there is a `Container` component here, should the `className` prop be applied to both  `Container` and `StyledCheckbox` ?